daliuge-translator/dlg/dropmake/web/main.js

```diff
@@ -421,8 +423,8 @@ function fillOutSettings() {
     console.debug("filling out settings, GET Errors and Cors warning from Url check")
     deployMethodsArray.forEach(async element => {
         const urlReachable = await checkUrlStatus(element.url);
-        const directlyAvailableMethods = await checkUrlSubmissionMethods(element.url + '/api/submission_method');
-        const translatorAvailableMethods = await checkUrlSubmissionMethods(getCurrentPageUrl() + '/api/submission_method?dlg_mgr_url=' + element.url);
+        const directlyAvailableMethods = await checkUrlSubmissionMethods(element.url + 'api/submission_method');
+        const translatorAvailableMethods = await checkUrlSubmissionMethods(getCurrentPageUrl() + 'api/submission_method?dlg_mgr_url=' + element.url);
```

Comment:
`getCurrentPageURL()` doesn´t append `/` to the end of the URL, so it looks like we would have `protocol/hostapi` instead of `protocol/host/api`. Is this what we want?